src/main/java/org/cascadebot/cascadebot/commands/management/FlagsCommand.java

```diff
@@ -0,0 +1,60 @@
+package org.cascadebot.cascadebot.commands.management;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Guild;
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.data.objects.Flag;
+import org.cascadebot.cascadebot.data.objects.GuildData;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.FormatUtils;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class FlagsCommand implements ICommandMain {
```

Comment:
why do we need this command? If they donated to the bot they should now what they're getting. Flags only really serve as a back end for donation stuff

Reply:
You're probably right. My thought behind it is that I would like some sort of way for admins to see what the server has

Reply:
I mean i could see how that'd be nice, but we need to add like descriptions to them so users acctuly know what they do.

Reply:
Agreed, I have a way I think I can do that nicely in Lang. Let's get this merged for now

Reply:
One thing first though